src/main.rs

```diff
@@ -64,28 +56,57 @@ fn main() {
             return;
         }
     };
-    thread::spawn(move || loop {
```

Comment:
any reason you took out the threading?

Reply:
it wasnt working, debugging rn might add it back later